lib/numbers.md

```diff
@@ -5,20 +5,23 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What are the different kinds of numbers in Ruby?
 
-Replace me with your response...
+In Ruby there are integers and floats.  An integer is any whole number, i.e. 1, 99, 0, -7.  A float is any number with a decimal, i.e. 1.875, 0.0, -3.33.
 
 # What are some common operations and comparisons you would perform on numbers?
 
-Replace me with your response...
+Most common operations would be to: add, subtract, multiply and divide.  The most common comparisons would be less than, greater than, equal to, not equal to,
+less than or equal to, and greater than or equal to.
 
 # What is the difference between the `+` operation on a number versus on a String?
 
-Replace me with your response...
+When working with numbers '+' will add two numbers together and return thier sum.  When working with strings, '+' will add two strings togeter - i.e. "Hello" + "World" would
+return "HelloWorld".
 
 # If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? If yes, why? If not, how would you make it work?
 
-Replace me with your response...
+No, it would not work.  In order for a mathematical operation to work on a string, the string must first be transformed into an integer using '.to_i'.
 
 # What is the purpose of the `times` operation? Is that the same as `*`?
 
-Replace me with your response...
+The '.times'method is not the same as '*'.  While '*' can be used to multiply strings, it will return the result as one long string.  The '.times' method will execute
+a command x number of times and will return the result each as its own string.  
```

Comment:
This is correct:

> The '.times' method will execute a command x number of times

But this is unclear:

> and will return the result each as its own string

Is the result of the `times` operation always a string? What are some examples of Ruby code using `times`?

Also, how would you write Ruby code to accomplish the following task: Create a variable `account_balance` and set it to the number `0`. Now execute a block of code 5 times, where each time it adds `5` to `account_balance`.